test/dialect/AIE/switchbox-ve2802.mlir

```diff
@@ -83,7 +83,6 @@ module {
     %01 = aie.tile(0, 1) // mem tile
     aie.switchbox(%01) {
       aie.connect<South: 0, South: 0> // Feedback OK
-      aie.connect<South: 0, South: 2> // Bounce OK
```

Comment:
from specs, connecting the same bundle with difference channel indices is not allowed in memtile